### DISCUSSION TITLE
nice slugs for sub pages

### DISCUSSION THREAD

**Discussion by ryeones** [velsa/notehost#51] [Q&A]
hi everyone, i am currently hosting my digital garden on notion using note host, and a very happy with how it works! one thing i wanted to ask was that in order to have nice slugs for **every single page**, does this mean i would have to write code for every single one of them or is there a more efficient way? sorry if this is a really basic question as i have zero coding background!
<img width="651" alt="image" src="https://github.com/user-attachments/assets/e1194971-e0ae-4408-917a-bcd21e5443a3">

**Comment by DudeThatsErin** [comment]
You have to do this for every single page.
